.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Pull request details
+
+## List of related issues or pull requests
+
+Refs: #ISSUE_NUMBER
+
+
+## Describe the changes made in this pull request
+
+
+## Instructions to review the pull request
```

Comment:
I prefer to not shout big captions at whoever makes the pull request, just boldface is enough

```suggestion
**List of related issues or pull requests**

Refs: #ISSUE_NUMBER

**Describe the changes made in this pull request**

**Instructions to review the pull request**
```